runtime/src/main/java/org/corfudb/util/serializer/ICorfuHashable.java

```diff
@@ -0,0 +1,10 @@
+package org.corfudb.util.serializer;
+
+/** Represents an object which the serializer may hash.
+ * Created by mwei on 7/16/17.
+ */
+public interface ICorfuHashable {
+
+    /** Generate a hash for this object to be used by Corfu. */
+    byte[] generateCorfuHash();
```

Comment:
Does the method name need to have Corfu in it. Is there a significance to that ?